Assets/Scripts/StaminaBar.cs

```diff
@@ -24,20 +27,13 @@ public class StaminaBar : MonoBehaviour {
 	}
 
     // Move is called each step of the players movement
-    // Returns state of stamina bar (crash or not);
+    // Returns if player can move;
     public bool move()

```

Comment:
This function might not be working as intended, I tried to play test and the player is unable to move at all [https://media.giphy.com/media/jR5hocXLGSd44/giphy.gif](url)